fixtures/coverall/tests/test_generated_bindings.rs

```diff
@@ -3,5 +3,5 @@ uniffi::build_foreign_language_testcases!(
     "tests/bindings/test_coverall.kts",
     "tests/bindings/test_coverall.rb",
     "tests/bindings/test_coverall.swift",
-    "tests/bindings/test_handlerace.kts",
+    //"tests/bindings/test_handlerace.kts",
```

Comment:
Why is this commented out now?

Reply:
Oops, that was just used while developing, will add it again.